### PR TITLE
Document pip.installed pkgs parameter

### PR DESCRIPTION
**Pull request by sblaisot**
### What does this PR do?

[`pip.installed` documentation](https://docs.saltproject.io/en/latest/ref/states/all/salt.states.pip_state.html#salt.states.pip_state.installed) lack documentation for the pkgs parameter to install multiple python modules at the same time.

This PR adds documentation for the `pkgs` parameter of `pip.installed` state.

### What issues does this PR fix or reference?

None

### Previous Behavior

pip.installed pkgs parameter undocumented

### New Behavior

pip.installed pkgs parameter documented

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**
- [x] Docs
- [N/A] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [N/A] Tests written/updated

### Commits signed with GPG?

No

### CODE REVIEW

**Review by barbaricyawps — APPROVED** [review]
@nicholasmhughes , this PR looks great to me. I made a very minor style suggestion, but I've approved it. Let me know if you want to make that change or not and when you're ready I'll get someone to merge this in.

**Review by barbaricyawps — DISMISSED** [review]
@sblaisot , whoops. I accidentally tagged Nick Hughes instead, but this PR looks good and I'm approving it. You can decide if you want to implement my style suggestion or not. If you don't want to, just let me know and we can merge this in.

File: salt/states/pip_state.py
Comment (sblaisot) [reply salt/states/pip_state.py:460]: OK, i will update it but this sentence has been copy/pasted from pkg.installed documentation so this should be updated there too to keep it coherent throughout all salt documentation

Comment (sblaisot) [reply salt/states/pip_state.py]: @barbaricyawps 

I updated this PR to match Google Developer's Style Guide as sugested.

I cannot find again similar sentence in `pkg.installed` so I should have dreamt of it (???)

**Comment by sblaisot** [comment]
@barbaricyawps I updated this PR to match Google Developer's Style Guide as sugested.. I think this can be merged now.

**Review by barbaricyawps — DISMISSED** [review]
Looks good! I've approved.

File: salt/states/pip_state.py
Comment (twangboy) [inline salt/states/pip_state.py:444]: Wouldn't you want to indent this whole section, starting with `Example` so that it falls under the `name` argument? It's just weird to see this indentation and then see the `pkgs` parameter right after.

File: salt/states/pip_state.py
Comment (twangboy) [inline salt/states/pip_state.py:464]: Same thing here, all this should be indented under the `pkgs` argument.

**Review by twangboy — CHANGES_REQUESTED** [review]

File: salt/states/pip_state.py
Comment (sblaisot) [reply salt/states/pip_state.py:464]: I only reproduced what was previously done, I though this was the current documentation style

File: salt/states/pip_state.py
Comment (sblaisot) [reply salt/states/pip_state.py:444]: This is the current state of the documentation: https://docs.saltproject.io/en/latest/ref/states/all/salt.states.pip_state.html#salt.states.pip_state.installed

**Comment by sblaisot** [comment]
@twangboy Both comments have been addressed.

I also want to share my extenal contributor feelings. Contributing to salt make me feel a real pain.

For a small documentation PR like this one, documenting a feature lacking documentation that has become essential now that onedir is the only package format available, this is:
- more than two monthes without any activity
- the need to ping someone on your slack channel to have this reviewed
- change requests unrelated to the initial object of this PR
- changes that could have been done within seconds by someone in the core team during merge
- more review after review and approval

IMHO, such a small documentation PR should have been merged within a week at most.

**Comment by Ch3LL** [comment]
@sblaisot apologies you have had some frustrations with getting your PR merged in. Agreed this process should be more simple, especially for a small change. Our review  process is the PR is auto assigned to a developer for review. They will  review the PR and ensure if another person needs to review the code as well. Going forward we will track that all new PRs will get a comment, a review OR merged within a week, so this does not occur in the future. This of course does not solve previous PRs. We have a backlog PR process that we attempt to do to bring down the backlog. This was done a few months back, before the chaos of trying to get 3006.0/3006.1 out  and I think we need to schedule out another sprint to tackle the backlog and do this regularly. I'm hoping with tracking to ensure new PRs are not lost and periodic sprints to get the PR backlog down will help ease these frustrations. 

@nicholasmhughes I'm not sure the details of the review inconsistencies across other PRs, but lets get this one figured out. @twangboy can you respond to @sblaisot 's comments?

**Review by twangboy — APPROVED** [review]
@sblaisot Thank you for making those changes. It makes the documentation more understandable, which I think is the point.

Sorry I took so long for the initial review

**Pull request merged**
